pavelib/assets.py

```diff
@@ -447,6 +449,10 @@ def update_assets(args):
         '--watch', action='store_true', default=False,
         help="Watch files for changes",
     )
+    parser.add_argument(
```

Comment:
@benpatterson Thank you so much for looking at this. We have a *lot* of confused users who get collectstatic failures that they can't debug because the results are piped to ``/dev/null``. I reported this ticket to change this behavior: https://openedx.atlassian.net/browse/FEDX-152

My original idea was to output everything to the console but it turns out that is too verbose for ordinary use cases. My revised idea is that if the user specifies ``--debug`` then we shouldn't pipe the results and should instead output them directly. I would also like to see the Ansible scripts then specify ``--debug`` too so that it is possible to understand a sandbox build failure just from the console log. Would you mind incorporating all of these thoughts into this PR?